openapi/io-functions-fast-login.yaml

```
@@ -0,0 +1,499 @@
+          description: The method of the endpoint called by IO app
+          required: true
+          schema:
+            $ref: '#/components/schemas/LollipopMethod'
+        - name: x-pagopa-lollipop-original-url
+          in: header
+          description: The url of the endpoint called by IO app
+          required: true
+          schema:
+            $ref: '#/components/schemas/LollipopOriginalURL'
+        - name: x-pagopa-lollipop-auth-jwt
+          in: header
+          description: Auth token to retrieve the SPID/CIE User identity from IO
+          required: true
+          schema:
+            $ref: '#/components/schemas/LollipopJWTAuthorization'
+        - name: x-pagopa-lollipop-assertion-ref
+          in: header
+          description: The id of the assertion to retrieve
+          required: true
+          schema:
+            $ref: '#/components/schemas/LollipopAssertionRef'
+        - name: x-pagopa-lollipop-assertion-type
+          in: header
+          description: The assertion type (SAML / OIDC)
+          required: true
+          schema:
+            $ref: '#/components/schemas/AssertionType'
+        - name: x-pagopa-lollipop-user-id
+          in: header
+          description: User's fiscal code.
+          required: true
+          schema:
+            $ref: '#/components/schemas/FiscalCode'
+        - name: x-pagopa-lollipop-public-key
+          in: header
+          description: User's public key
+          required: true
+          schema:
+            $ref: '#/components/schemas/LollipopPublicKey'
+        - name: signature-input
+          in: header
+          description: The signature input, needed to verify the `signature` header
+          required: true
+          schema:
+            $ref: '#/components/schemas/LollipopSignatureInput'
+        - name: signature
+          in: header
+          description: >-
+            The signature of the HTTP request, signed by the client with its
+            private key.
+          required: true
+          schema:
+            $ref: '#/components/schemas/LollipopSignature'
+      responses:
+        '200':
+          description: Success response
+          content:
+            application/json:
+              schema:
+                $ref: '#/components/schemas/FastLoginResponse'
+        '400':
+          description: Bad request
+          content:
+            application/json:
+              schema:
+                $ref: '#/components/schemas/BadRequest'
+        '401':
+          description: Unauthorized
+          content:
+            application/json:
+              schema:
+                $ref: '#/components/schemas/Unauthorized'
+        '500':
+          description: Server error
+          content:
+            application/json:
+              schema:
+                $ref: '#/components/schemas/InternalServerError'
+  /v1/logout:
+    post:
+      operationId: logoutFromIOApp
+      summary: Logout User from IO App
+      requestBody:
```

the body has to be required
```suggestion
      requestBody:
        required: true
```